oe_content.module

```diff
@@ -74,3 +75,31 @@ function oe_content_rdf_entity_access(EntityInterface $entity, $operation, Accou
   $provenance_uri = \Drupal::config('oe_content.settings')->get('provenance_uri');
   return $entity->get('provenance_uri')->value === $provenance_uri ? AccessResult::neutral() : AccessResult::forbidden();
 }
+
+/**
+ * Implements hook_entity_load().
+ */
+function oe_content_taxonomy_term_load(array $entities) {
+  // Taxonomy terms have a status field for handling publishing state. We need
+  // to default to TRUE for all terms.
```

Comment:
We have no choice now other than adding this to all loaded terms since the `rdf_taxonomy` module takes completely over (storage can be swapped on entity type, after all). Shall we maybe add a comment explaining that we will refactor this by switching from taxonomy to a custom entity type?

Reply:
Use `hook_ENTITY_TYPE_storage_load()` instead of `hook_ENTITY_TYPE_load()`, because that also is caching the altered value. `hook_ENTITY_TYPE_load()` runs every time regardless if the items were retrieved from cache or not.